.github/workflows/build-images.yml

```diff
@@ -315,7 +315,7 @@ jobs:
     - name: Build and optionally push v2 server image
       if: ${{ inputs.v2_vertica_image == '' }}
       run: |
-        export VERTICA_IMG=${{ steps.v2_vertica_image.outputs.value }}
+        export VERTICA_IMG=verticadocker/vertica-k8s-private:latest-master
```

Comment:
Not sure if this is still WIP, but we must honour the image that's passed into e2e.yml. We should only use latest-master if it the e2e.yml was invoked through a push, PR or workflow dispatch and v2 image was empty.

Reply:
Yes, It's ready for review now.